app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,168 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+// require rails_bootstrap_forms
+
+
+body {
+  background-color: #fafafa;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  padding-top: 70px;
+}
+
+// =========================================================
+/* ## NAVBAR CUSTOMISATIONS ## */
+// =========================================================
+
+.navbar-brand {
+  a {
+    color: #125688;
+  }
+
+  img {
+    width: 25px;
+    height: auto;
+  }
+}
+
+.navbar-default {
+  background-color: #fff;
+  height: 54px;
+  .navbar-nav li a {
+    color: #125688;
+  }
+}
+
+.navbar-container {
+  width: 70%;
+  margin: 0 auto;
+}
+
+// =========================================================
+/* ## POST CUSTOMISATIONS ## */
+// =========================================================
+
+.posts-wrapper {
+  padding-top: 40px;
+  margin: 0 auto;
+  max-width: 642px;
+  width: 100%;
+}
+
+.post {
+  background-color: #fff;
+  border-color: #edeeee;
+  border-style: solid;
+  border-radius: 3px;
+  border-width: 1px;
+  margin-bottom: 60px;
+  .post-head {
+    flex-direction: row;
```

Comment:
Properties should be ordered color, flex-direction, font-size, height, line-height, padding-left, padding-right, padding-top